src/libraries/Microsoft.PowerFx.Core/Functions/DefaultResourceElement.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.PowerFx.Core.Functions
+{
+    public abstract class BaseResourceElement
+    {
+        public abstract bool IsUri { get; }
+
+        public virtual Uri Uri => new Uri($"appres://blobmanager/{Handle.Handle}");
```

Comment:
>appres [](http://example.com/codeflow?start=44&length=6)

Minor, but we may want to use some other scheme here. "appres" refers to an "app resource", which makes sense in Power Apps (we have an "app", after all), but not necessarily in other places (e.g., CLI). Suggestion: blob:

Reply:
I'll make it configurable & fxres by default